common/src/main/java/com/wynntils/core/managers/ManagerRegistry.java

```diff
@@ -103,7 +89,7 @@ public static void removeDependency(Feature dependant, Class<? extends Model> de
         updateManagerState(dependency);
     }
 
-    public static void removeAllFeatureDependency(Feature dependant) {
+    public static void removeAllDependency(ModelDependant dependant) {
         for (Class<? extends Manager> manager : MODEL_DEPENDENCIES.keySet()) {
```

Comment:
This is not correct. The class should be of ModelDependant, and the name should reflect that.

Reply:
I was going to update this method as well in this pr.